app/templates/direct-award/save-search.html

```diff
@@ -43,8 +43,16 @@
         <div class="dmspeak">
           <h2 class="heading-xmedium">Choose where to save your search</h2>
         </div>
+        {% if save_search_selection_error %}
+        <p class="validation-message" id="error-save-search-selection">
+          {{ save_search_selection_error }}
+        </p>
+        {% endif %}
         
         <fieldset>
+          {% if save_search_selection_error %}
+          <div class="validation-wrapper">
```

Comment:
Can we have the `<div>` always be in the document, but the class added only if the condition is met? This means you only need one `{% if %}` block.

Reply:
Ahh, that could be quite smart if possible. Do you think it would impact on reliability?

Reply:
@galund - this has been changed for you.